implementations/c/ockam/transport/posix_socket/socket.h

```diff
@@ -14,6 +14,8 @@
 #include "ockam/transport/impl.h"
 #include "socket.h"
 
+// TODO: add ockam_ prefix to types declared here. Review which of them indeed need to be public.
```

Comment:
let's try to note these types of TODOs in github issues, otherwise there is high likely hood that they will get missed.